lynx_description/launch/load_urdf.launch.py

```diff
@@ -91,6 +82,22 @@ def generate_launch_description():
         description="Add namespace to all launched nodes.",
     )
 
+    use_joint_state_publisher = LaunchConfiguration("use_joint_state_publisher")
+    declare_use_joint_state_publisher_arg = DeclareLaunchArgument(
+        "use_joint_state_publisher",
+        default_value="False",
+        description="Flag enabling joint_state_publisher to publish information about joints positions. Should be false when there is a controller that sends this information.",
+        choices=["True", "true", "False", "false"],
+    )
```

Comment:
I someone just would like test robot_state_publisher and use this single launch file the wheels will not display correctly in rviz without `joint_state_publisher` node. 
```suggestion
    use_joint_state_publisher = LaunchConfiguration("use_joint_state_publisher", default="True")
```